spec/sidekiq/evss/disability_compensation_form/submit_uploads_spec.rb

```diff
@@ -26,17 +27,18 @@
   let(:form526_job_status) { create(:form526_job_status, :retryable_error, form526_submission: submission, job_id: 1) }
   let(:upload_data) { [submission.form[Form526Submission::FORM_526_UPLOADS].first] }
 
+  let(:file) { Rack::Test::UploadedFile.new('spec/fixtures/files/sm_file1.jpg', 'image/jpg') }
+  let!(:attachment) do
+    sea = SupportingEvidenceAttachment.new(guid: upload_data.first['confirmationCode'])
+    sea.set_file_data!(file)
+    sea.save!
+    sea
+  end
```

Comment:
There was a lot of duplication in this file and I just ended up adding to it so I went ahead and refactored the existing tests to share all of this (an also return the attachment, this was returning `true` because the last line is `save!` which isn't very useful for tests.

PS because of the way file data works for attachments we have to have this rigamorale in the tests, kind of a code smell but way outside the scope of what I'm doing here